Description: Remove routine check-man-tools

check-man-tools fails to detect that the packages are installed.
Has been reported upstream at:
https://lore.kernel.org/linux-trace-devel/CADVatmNE_Bm6Gv9ArNsTQhJME9y7gbnExFf+iy_=68CSKKZ+Xw@mail.gmail.com/T/#u

---

--- libtracefs-0.0~git20201221.74d7273.orig/Documentation/Makefile
+++ libtracefs-0.0~git20201221.74d7273/Documentation/Makefile
@@ -153,14 +153,14 @@ do-install-man: man
 		$(INSTALL) -d -m 755 $(DESTDIR)$(man3dir); \
 		$(INSTALL) -m 644 $(OUTPUT)*.3 $(DESTDIR)$(man3dir);
 
-install-man: check-man-tools man do-install-man
+install-man: man do-install-man
 
 do-install-html: html
 	$(call QUIET_INSTALL, Documentation-html) \
 		$(INSTALL) -d -m 755 $(DESTDIR)$(htmldir); \
 		$(INSTALL) -m 644 $(OUTPUT)*.html $(DESTDIR)$(htmldir);
 
-install-html: check-man-tools html do-install-html
+install-html: html do-install-html
 
 uninstall: uninstall-man uninstall-html
 
